cypress/integration/supervision/dashboard/dashboard.spec.js

```diff
@@ -1,15 +1,5 @@
-before(function fetchUser () {
-  cy.request({
-    method: 'POST',
-    url: '/',
-    form: true,
-    body: {
-      email: 'case.manager@opgtest.com',
-      password: 'Password1',
-      submit: 'Sign In',
-      'OPG-Bypass-Membrane': 1
-    }
-  });
+before(function loginAsCaseManager () {
```

Comment:
Is there a reason for using a named function here? This could be written more simply as `before(() => cy.loginAs('Case Manager');`.